ush/retrieve_data.py

```diff
@@ -897,10 +954,17 @@ def parse_args(argv):
     # Required
     parser.add_argument(
         "--anl_or_fcst",
-        choices=("anl", "fcst"),
+        choices=("anl", "fcst", "obs", "None"),   # Added 'None' for generic packages copied from local disk or remote url
```

Comment:
Since this flag has been change to _not_ be required, I don't think that the string option "None" should be included. The `cla.anl_or_fcst` variable will be `None` if the user doesn't provide this flag. If you do think it should be required, then we do need a "none" option, but I'd go with all lower case for consistency -- it would be saved as a string not a `None` object and that would be confusing.

Reply:
renamed to "file_set"